src/main/java/hanieum/conik/application/company/CompanyService.java

```diff
@@ -0,0 +1,73 @@
+package hanieum.conik.application.company;
+
+import hanieum.conik.application.company.provided.CompanyFinder;
+import hanieum.conik.application.company.provided.CompanyRegister;
+import hanieum.conik.application.company.required.CompanyRepository;
+import hanieum.conik.domain.company.Company;
+import hanieum.conik.domain.company.dto.CompanyRegisterRequest;
+import hanieum.conik.domain.company.exception.CompanyErrorType;
+import hanieum.conik.domain.company.exception.CompanyException;
+import hanieum.conik.domain.member.shared.Email;
+import hanieum.conik.global.adapter.s3.dto.ImageUploadRequest;
+import hanieum.conik.global.adapter.s3.dto.ReadPreSignedUrlResponse;
+import hanieum.conik.global.application.required.BucketClient;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.validation.annotation.Validated;
+
+import java.util.List;
+
+@Slf4j
+@Service
+@Transactional
+@Validated
+@RequiredArgsConstructor
+public class CompanyService implements CompanyFinder, CompanyRegister {
+
+    private final BucketClient bucketClient;
+    private final CompanyRepository companyRepository;
+
+    @Override
+    @Transactional(readOnly = true)
+    public List<Company> findAllCompanies() {
+        return companyRepository.findAll();
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public Company findCompanyById(Long companyId) {
+        return companyRepository.findById(companyId).orElseThrow(() -> new CompanyException(CompanyErrorType.COMPANY_NOT_FOUND));
```

Comment:
너무 코드가 길어지면 한 눈에 보기 힘들어서, 
```suggestion
        return companyRepository.findById(companyId)
         .orElseThrow(() -> new CompanyException(CompanyErrorType.COMPANY_NOT_FOUND));
```
이런 식으로 작성해도 좋을 것 같아요